modules/manage/pages/cluster-maintenance/manage-throughput.adoc

```diff
@@ -43,48 +45,200 @@ The properties for broker-wide throughput quota balancing are configured at the
+
+To more easily manage multiple quotas, you can use the `cluster quotas describe` and xref:reference:rpk/rpk-cluster/rpk-cluster-quotas-import.adoc[`cluster quotas import`] commands to do a bulk export and update.
+
+For example, to export all client quotas in JSON format:
+
+[,bash]
+----
+rpk cluster quotas describe --format json
+----
+
+`rpk cluster quotas import` accepts the output string from `rpk cluster quotas describe --format <json/yaml>`:
+
+[,bash]
+----
+rpk cluster quotas import --from '{"quotas":[{"entity":[{"name":"foo","type":"client-id"}],"values":[{"key":"consumer_byte_rate","values":"12123123"}]},{"entity":[{"name":"foo-","type":"client-id-prefix"}],"values":[{"key":"producer_byte_rate","values":"12123123"},{"key":"consumer_byte_rate","values":"4444444"}]}]}'
+----
+
+You may also save the JSON or YAML output to a file and pass the file path in the `--from` flag.
+
+[[migrate]]
+=== Migrate cluster configuration quotas to Kafka API-based quotas
+
+. Use xref:reference:rpk/rpk-cluster/rpk-cluster-config-get.adoc[`rpk cluster config get`] to view current client quotas managed with cluster configuration. The following example shows how to retrieve the `kafka_client_group_byte_rate_quota` for two groups of producers:
++
+[,bash]
+----
+rpk cluster config get kafka_client_group_byte_rate_quota
+
+----
++
+[,bash,role=no-copy]
+----
+"kafka_client_group_byte_rate_quota": [ 
+  { 
+    "group_name": "group_1", 
+    "clients_prefix": "producer_group_alone_producer", 
+    "quota": 10240 
+  }, 
+  { "group_name": "group_2", 
+    "clients_prefix": "producer_group_multiple", 
+    "quota": 20480 
+  }, 
 ]
 ----
+
+. Each client quota cluster property corresponds to a quota type in Kafka. Check the corresponding `rpk` arguments to use when setting the new quota values:
++
+|===
+| Cluster configuration property | `rpk cluster quotas` arguments
+
+| xref:reference:properties/cluster-properties.adoc#target_quota_byte_rate`[`target_quota_byte_rate`]                      
+| `--default client-id --add producer_byte_rate=<producer-byte-rate-value>`
+
+| xref:reference:properties/cluster-properties.adoc#target_fetch_quota_byte_rate[`target_fetch_quota_byte_rate`]                
+| `--default client-id --add consumer_byte_rate=<consumer-byte-rate-value>`
+
+| xref:reference:properties/cluster-properties.adoc#kafka_admin_topic_api_rate[`kafka_admin_topic_api_rate`]                  
+| `--default client-id --add controller_mutation_rate=<controller-mutation-rate-value>`
+
+| xref:reference:properties/cluster-properties.adoc#kafka_client_group_byte_rate_quota[`kafka_client_group_byte_rate_quota`]          
+| `--name client-id-prefix=<prefix> --add producer_byte_rate=<group-producer-byte-rate-value>`
+
+| xref:reference:properties/cluster-properties.adoc#kafka_client_group_fetch_byte_rate_quota[`kafka_client_group_fetch_byte_rate_quota`]    
+| `--name client-id-prefix=<prefix> --add consumer_byte_rate=<group-consumer-byte-rate-value>`
+
```

Comment:
@kbatuigas: Suggest removing all these links and adding a reminder note about how they're deprecated in 24.2